topiary-queries/queries/nix.scm

```diff
@@ -0,0 +1,221 @@
+;; Nix Formatter based on RFC 166
+
+;; Nix Formatter based on RFC 166
+
+; Leaf nodes
+[
+  (string_expression)
+  (integer_expression)
+  (float_expression)
+  (uri_expression)
+  (path_expression)
+  (hpath_expression)
+  (spath_expression)
+] @leaf
+
+; Comments
+(comment) @comment
+
+; Convert single-line /* */ comments to #
+(comment) @convert_to_hash_comment
+
+; Preserve multiline comments
+(comment) @preserve_multiline_comment
+
+; Append space after specific tokens
+":" @append_space
+"=" @append_space
+"," @append_spaced_softline
+
+; Handle indentation and line breaks
+; Define softline breaks and indentation starts/ends
+"{" @append_spaced_softline @append_indent_start
+"}" @prepend_spaced_softline @prepend_indent_end
+"[" @append_spaced_softline @append_indent_start
+"]" @prepend_spaced_softline @prepend_indent_end
+
+; Functions
+
+(function_expression
+  formals: (formals
+    "(" @append_spaced_softline @append_indent_start
+    (formal) @formal_param
+    ")" @prepend_spaced_softline @prepend_indent_end
```

Comment:
```suggestion
    "{" @append_spaced_softline @append_indent_start
    (formal) @formal_param
    "}" @prepend_spaced_softline @prepend_indent_end
```

Ah found it. `formals` was a red herring, it's that it doesn't expect `(` to be a children of `formals` - I guess you meant `{` instead. This seems to work.